.env.test

```diff
@@ -0,0 +1,2 @@
+CREATE_EXCLUDE_PARAMS=""
+JOIN_EXCLUDE_PARAMS="test1, test2"
```

Comment:
The exclude params list should not contain a space - it's comma-separated, not "comma + space"-separated.

Reply:
(This also needs to be updated in the README)